SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the SunPy software library
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. github account, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombusperson
+
+The SunPy project is ultimately run by the whole community: users, developers, and members of the wider solar physics and scientific software communtiy, for the benifit of all.
+This document descibes how the project is run through delegation of this authority, primarily to the Steering Committee and the Project Members.
+Wherever possible all descisions taken should be done by consensus, considering input from both active contributors and the wider community.
```

Comment:
```suggestion
Wherever possible all descisions taken should be done by consensus, considering input from both active developers and the wider community.
```
I think it's best to stick to one of "developers" or "contributors", unless there's a difference between them and we need both terms?

Reply:
👍